core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -458,7 +458,7 @@ private class LiveStage extends LiveEntity {
 
 }
 
-private class LiveRDDPartition(val blockName: String) {
+private class LiveRDDPartition(val blockName: String, rddLevel: StorageLevel) {
```

Comment:
It would help me a bit if this was called 'requestedStorageLevel' and in RDDPartitionInfo it was called 'effectiveStorageLevel'.  I guess its not worth changing RDDPartitionInfo, but maybe just a comment along those lines.

Reply:
Sure.